packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.jsx

```diff
@@ -37,8 +44,24 @@ const propTypes = {
 
 class EmbeddedContentConsumer extends React.Component {
   componentDidMount() {
+    // Merging the iframe options props
+    const frameOptions = { ...this.props.options };
+    if (frameOptions.iframeAttrs == null) {
```

Comment:
This section of code should be able to be moved into the if statement directly below it.

Talked with Kol and it looks like we only care about this check/set if we are calling into the `if (this.props.title)` statement. If we don't call into that statement then we can just keep `frameOptions.iframeAttrs == null` and don't need to set it to `frameOptions.iframeAttrs = {};`.